src/coreclr/jit/lowerxarch.cpp

```diff
@@ -9665,6 +9664,12 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                         {
                             GenTreeHWIntrinsic* childNode = op1->AsHWIntrinsic();
 
+                            if (childNode->Op(1)->OperIsLong())
```

Comment:
What guarantees the child node is an intrinsic that has any operands?

Reply:
Moved this bailout back to where it was. The handling for CreateScalar should be the same as for CreateScalarUnsafe.